Sketch/SketchController.m

```diff
@@ -27,7 +34,7 @@ - (void) applicationDidFinishLaunching: (NSNotification *)notif
 	// base located in the bottom left.
 	//[mainItem setFlipped: NO];
 	[mainItem setSize: NSMakeSize(500, 400)];
-	[mainItem setLayout: [ETBezierFreeLayout layout]];
+	[mainItem setLayout: [ETBezierFreeLayout layoutWithObjectGraphContext: [self objectGraphContext]]];
```

Comment:
If there's an object graph specific to the document's UI, can the item factory be used anywhere the object graph needs to be accessed? If so, I should be able to access the graph within ETBezierHandler's initializer like so `[[ETLayoutItemFactory factory]  objectGraphContext]`, correct?